<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QMainWindow" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>设定风速：</string>
    </property>
   </widget>
   <widget class="QLabel" name="model_of_main">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_goaltemp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_currenttemp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>从控机系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="cost">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>空调模式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>当前温度：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电源</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>费  用：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_wind">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>风速</string>
    </property>
   </widget>
   <widget class="QLabel" name="wind_speed">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>目标温度：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_temp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温度+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_temp_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温度-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_mode">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退房</string>
    </property>
   </widget>
   <widget class="QLabel" name="wind_speed_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>当前风速：</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>on</sender>
   <signal>clicked()</signal>
   <receiver>User</receiver>
   <slot>state_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_mode</sender>
   <signal>clicked()</signal>
   <receiver>User</receiver>
   <slot>mode_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_wind</sender>
   <signal>clicked()</signal>
   <receiver>User</receiver>
   <slot>speed_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_temp</sender>
   <signal>clicked()</signal>
   <receiver>User</receiver>
   <slot>tem_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_temp_2</sender>
   <signal>clicked()</signal>
   <receiver>User</receiver>
   <slot>tem_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>User</receiver>
   <slot>checkout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>state_change()</slot>
  <slot>mode_change()</slot>
  <slot>speed_change()</slot>
  <slot>tem_up()</slot>
  <slot>tem_down()</slot>
  <slot>checkout()</slot>
 </slots>
</ui>
